src/components/time-entries/TimeEntries.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/destructuring-assignment */
```

Comment:
Normally, you would want to resolve this in ESLint or the page, but very rarely like this

Reply:
I know, I discussed this with Sabine and we came to the conclusion that at this stage we don't change the `props.variable`, because ESLint didn't work previously. In a next project I will take this feedback with me.